### ISSUE TITLE
Not available in Ruby 2.6, any further plan?

### ISSUE THREAD

**Issue by tan-wei** [jarmo/win32screenshot#27]
It seems that `win32scrrenshot` is not available now because `ffi-1.9.25` can not installed in Ruby 2.6. So do we have any plan on this?
Thanks very much!

**Comment by tan-wei** [comment]
Sorry for I forget upload the log:
```
ERROR:  Error installing win32screenshot:
        The last version of ffi (~> 1.9.0) to support your Ruby & RubyGems was 1.9.25. Try installing it with `gem install ffi -v 1.9.25` and then running the current command again
        ffi requires Ruby version >= 2.0, < 2.6. The current ruby version is 2.6.0.0.
```

**Comment by jarmo** [comment]
I don't have a Windows machine nearby to test it out, but would win32screenshot work if its ffi dependency would be upgraded to support Ruby 2.6+? Can you test it out by cloning this repository, upgrading ffi and running its specs (and looking at the images done by tests just in case)?

**Comment by tan-wei** [comment]
No problem, I'll try it.

**Comment by tan-wei** [comment]
Unfortunately, I think my Windows machine can not install `win32screenshot` now. I find that `RAutomation` should be installed first, and it also depends on `ffi 1.9.25`. It's not the problem until I see the sentence:
```
Currently only 32bit Ruby is supported. 64bit versions do not work at all!
```
Currently, I use Ruby 2.6.3-x64-Windows. 😭

**Comment by jarmo** [comment]
You're welcome to upgrade RAutomation to support x64 too ;)

**Comment by tan-wei** [comment]
LOL, I will try it in x86 till I install a x86 version Ruby in another machine. I'd better to use x86 version one because some native codes should run with my Ruby codes, they also only support x86 in fact. :-P

**Comment by jarmo** [comment]
Released 3.0.0, which should work with Ruby 2.6 too (only 32bit due to its dependency RAutomation!).
